website/sales/models/shift.py

```diff
@@ -49,7 +49,10 @@ class Meta:
     )
 
     managers = models.ManyToManyField(
-        MemberGroup, verbose_name=_("managers"), related_name="manager_shifts"
+        MemberGroup,
+        verbose_name=_("managers"),
+        related_name="manager_shifts",
+        blank=True,
```

Comment:
It does not make much sense to me that a shift cannot have managers?

Reply:
I thought it could be useful when, for example, a treasurer wants to open a shift (although I see they could just fill in the board). It could be useful for people that have the override permission

Reply:
I don't think we should do that, it's better to always be able to have at least one group of managers. That way there does not have to be a None check and it is clear who is responsible for it.

Reply:
I agree with that in fact